### ISSUE TITLE
Boutiques tasks crash when exit code file doesn't exist

### ISSUE THREAD

**Issue by glatard** [aces/cbrain#579]
```
[2016-12-22 11:39:56 EST] post_process() Exception raised while post processing results: Errno::ENOENT: No such file or directory @ rb_sysopen - .qsub.exit.PreFreeSurferPipelineBatch.123-1
[2016-12-22 11:39:56 EST] /home/glatard/code/cbrain/Bourreau/app/models/cluster_task.rb:746:in `block (2 levels) in post_process'
[2016-12-22 11:39:56 EST] /home/glatard/code/cbrain/Bourreau/app/models/cluster_task.rb:740:in `chdir'
[2016-12-22 11:39:56 EST] /home/glatard/code/cbrain/Bourreau/app/models/cluster_task.rb:740:in `block in post_process'
```

**Comment by natacha-beck** [comment]
@glatard just looking at this old issue is it still the case, if your code on #579 solved it we should integrate it on bourreau template. Can you give up an update on this issue.

**Comment by natacha-beck** [comment]
Tested it today and it was fixed by the following commit 26b17c2d1d6462a4513f2976972d1b2204f4126e
